### PR TITLE
Add flask-httpauth recipe

### PR DESCRIPTION
**Pull request by synapticarbors**

### CODE REVIEW

**Comment by synapticarbors** [comment]
There is a discrepancy between the license file in the dev repo of this project (MIT) and how it is reported to pypi (BSD). I've changed it to MIT to reflect the actual license file and raised an issue (miguelgrinberg/Flask-HTTPAuth#44) with the project owner miguelgrinberg

**Comment by synapticarbors** [comment]
The license issue has been resolved and the flask-httpauth author has updated the released version. I've updated the conda recipe accordingly.

**Comment by msarahan** [comment]
I don't understand what the current best practice is regarding the license file.  I thought that you needed it with MIT projects?  Is it included in the package somehow one way or another?

Have you pinged miguelgrinberg about being a maintainer here?  I guess we sometimes do that after the feedstock is created.

**Comment by synapticarbors** [comment]
@msarahan I asked miguelgrinberg in the following issue about being a co-maintainer:

https://github.com/miguelgrinberg/Flask-HTTPAuth/issues/44

Still waiting for a response.

File: recipes/flask-httpauth/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    - python
+    - flask
+
+test:
+  imports:
+    - flask_httpauth
+
+about:
+  home: http://github.com/miguelgrinberg/flask-httpauth/
+  license: MIT
```
Comment (jakirkham) [inline recipes/flask-httpauth/meta.yaml]: Can we please add a note to include a license file once we can? A link to issue ( https://github.com/miguelgrinberg/Flask-HTTPAuth/issues/45 ), would also be good.

Comment (synapticarbors) [inline recipes/flask-httpauth/meta.yaml]: I'll add this issue to the main feedstock once it gets merged out of staging.

**Comment by jakirkham** [comment]
Thanks @synapticarbors.

**Pull request merged**